_docs/guides/create-notes-occurrences.md

```diff
@@ -0,0 +1,251 @@
+---
+title: How to create notes and occurrences
+overview: Describes how to create notes and occurrences once you have a running Grafeas server.
+
+order: 15
+
+layout: docs
+type: markdown
+---
+
+Before you can create [notes](/_docs/concepts/what-is-grafeas/overview.md) or
+[occurrences](/_docs/concepts/what-is-grafeas/overview.md), you'll need a
+running Grafeas server, which you can set up following
+[these instructions](https://github.com/grafeas/grafeas/blob/master/docs/running_grafeas.md).
+The examples below assume that you have a local server running on port 8080.
+
+## Create a project
+
+All notes and occurrences must be stored in a project. Generally, you should
+create separate projects for notes and occurrences in order to make it easier to
+set up more fine-grained access control.
+
+To create a `provider_example` project to store notes in, run the following
+command:
+
+```bash
+curl -X POST http://localhost:8080/v1alpha1/projects \
+     -H "Content-Type: application/json" \
+     --data '{"name":"projects/provider_example"}'
+```
+
+Next, create an `occurrence_example` project:
+
+```bash
+curl -X POST http://localhost:8080/v1alpha1/projects \
+     -H "Content-Type: application/json" \
+     --data '{"name":"projects/occurrence_example"}'
+```
+
+## Using curl
```

Comment:
How about the following format:

## Creating a note

### Using curl

### Using Java

## Creating an occurrence

### Using curl

### Using Java